docs/2.0/docs/library/concepts/component-modules.md

```diff
@@ -0,0 +1,52 @@
+# Component Modules
```

Comment:
I don't love `component module` personally - What is a component?  It's a component of what?  Adjacent to the word "composite" it's a little clearer, but on its own I think the prior term "building block" was more easily understood.

Also - changing names is hard, not something we should do often, and we should make sure to overcommunicate with the rest of the team first.